src/features/sendEmail/SendEmail.tsx

```diff
@@ -0,0 +1,124 @@
+import {
+  Alert,
+  Box,
+  Button,
+  Snackbar,
+  TextField,
+  TextareaAutosize,
+  Typography,
+} from '@mui/material';
+import './SendEmail.css';
+import { useParams } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import { useSendEmailMutation } from '../../app/appSlice';
+
+export default function SendEmail() {
+  const { receiverAddress, protectedDataAddress } = useParams();
+
+  //RTK Mutation hook
+  const [sendEmail, result] = useSendEmailMutation();
+
+  //for textarea
+  const [value, setValue] = useState('');
+  //limited to 4096 by the SMS
+  const [charactersRemainingMessage, setCharactersRemainingMessage] =
+    useState(4096);
+
+  //for name et dataType
+  const [messageSubject, setMessageSubject] = useState('');
+  //limited to 78 by the SMS
+  const [charactersRemainingSubject, setCharactersRemainingSubject] =
+    useState(78);
+
+  //handle functions
+  const handleMessageSubjectChange = (event: any) => {
+    const inputValue = event.target.value;
+    setMessageSubject(inputValue);
+    setCharactersRemainingSubject(78 - inputValue.length);
+  };
+
+  const handleChange = (event: any) => {
+    const inputValue = event.target.value;
+    setValue(inputValue);
+    setCharactersRemainingMessage(4096 - inputValue.length);
+  };
+
+  const sendEmailHandle = () => {
+    if (!protectedDataAddress) return;
+    sendEmail({
+      emailSubject: messageSubject,
+      emailContent: value,
+      protectedData: protectedDataAddress,
+    });
+  };
+
+  //snackbar notification
+  const [open, setOpen] = useState(false);
+  const [success, setSuccess] = useState(false);
```

Comment:
create a ticket in the backlog to add a proper notification system in the whole app